python/katana/analytics/_betweenness_centrality.pyx

```diff
@@ -118,10 +115,16 @@ cdef class BetweennessCentralityPlan(Plan):
 
     @staticmethod
     def outer():
+        """
+        Parallelize outer-most iteration
```

Comment:
I thought we were using a single line for the comments, like """Parallelize outer-most iteration""" (and periods at the end). Minor only.

Reply:
I don't like the single line form, but it's not something I push other people on. Do you think we need to be consistent about it?

Reply:
If we can, great.  If not, not a big deal. Go ahead with the PR no need to block on that.

Reply:
Ok. I would actually prefer to consistently use the multiline form. I like how it sets out the documentation in the source, but I will look at style guides and what black does. For the moment, let's just be consistent within each file and we can create a global standard soon.